### PR TITLE
feat: chat with encrypted messages in groups, fix: edited posts

### PR DESCRIPTION
**Pull request by maxgithubprofile**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] The PR has self-explained commits history.

- [x] The code is mine or it's from somewhere with an Apache-2.0 compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.
- [x] If the code introduces new classes, methods, I provide a valid use case for all of them.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

**Comment by shpingalet007** [comment]
Not the best name for a branch - **`Crypto-in-groups,-read-crypto-messages`**. Consider reading [this article](https://deepsource.io/blog/git-branch-naming-conventions/) to get the idea.

## TLDR;
- Don't use special characters, e.g. commas, dots, question marks etc.
- Use hyphens as separators
- If there is an issue related to the pull, start branch name with its number

**Review by only1question — APPROVED** [review]

**Review by shpingalet007 — APPROVED** [review]
Quality checked
